pkg/cmd/cli/cmd/deploy.go

```diff
@@ -350,7 +350,7 @@ func (o DeployOptions) cancel(config *deployapi.DeploymentConfig) error {
 				continue
 			}
 
-			deployment.Annotations[deployapi.DeploymentCancelledAnnotation] = deployapi.DeploymentCancelledAnnotationValue
+			deployment.Annotations[deployapi.DeploymentCancelledAnnotation] = unversioned.Now().Format(time.RFC3339)
```

Comment:
compatibility goes both ways... if a new client sends a timestamp to an old server, will the old server be expecting `true` and fail?